### PR TITLE
Create Dockerfile

### PR DESCRIPTION
**Pull request by meiremans**
The dockerfile expects the cool-runings executable and a "tests.yml" file in the same directory as the dockerfile.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+FROM ubuntu:16.04
+# Install.
+RUN \
+  apt-get update && \
+  apt-get -y upgrade && \
+  apt-get install -y curl
+  
+# Add files.
+WORKDIR /home
+RUN curl -sSL https://get.haskellstack.org/ | sh
```
Comment (aviaviavi) [inline Dockerfile]: what's the purpose of downloading stack here? i suppose i haven't tried running the binary on a machine without it, but i didn't expect it to be needed

Comment (meiremans) [reply Dockerfile]: Hello, ok i will update the readme too.
And for the stack, indeed it is not necessary anymore. The reason it was there is that in my first dockerfile I compliled it when building the docker image. But it took to long imho so I switched to just coping the executable.
I will remove that line too.

**Comment by meiremans** [comment]
Done!

**Comment by aviaviavi** [comment]
Thanks for the changes. I take it you didn't mean to close this, correct? Reopening, will merge soon

**Comment by meiremans** [comment]
> Thanks for the changes. I take it you didn't mean to close this, correct? Reopening, will merge soon

No sorry, that was a mistake

**Pull request merged**